src/main/java/uk/gov/hmcts/reform/unspec/model/robotics/LitigiousParty.java

```diff
@@ -1,26 +1,27 @@
 package uk.gov.hmcts.reform.unspec.model.robotics;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Builder;
 import lombok.Data;
 
-import java.time.LocalDate;
-
 @Data
 @Builder
 public class LitigiousParty {
 
-    private String litigiousPartyType;
-    private String litigiousPartyName;
-    private RoboticsAddresses litigiousPartyAddresses;
-    private String litigiousPartyContactDX;
-    private String litigiousPartyContactTelephoneNumber;
-    private String litigiousPartyContactFaxNumber;
-    private String litigiousPartyContactEmailAddress;
-    private String litigiousPartyPreferredMethodOfCommunication;
-    private String litigiousPartyWelshTranslation;
-    private String litigiousPartyReference;
-    private String litigiousPartyDateOfService;
-    private String litigiousPartyLastDateForService;
-    private LocalDate litigiousPartyDateOfBirth;
-    private String litigiousPartySolicitorPartyOrganisationID;
+    @JsonProperty("ID")
+    private String id;
+    private String type;
+    private String name;
+    private RoboticsAddresses addresses;
```

Comment:
plural naming is actually misleading here, this isn't a list

Reply:
I would do RoboticsAddresses -> ContactAddress I think

Reply:
or just a thought... maybe it should be a list?

Reply:
there will be more addresses composed in this object.